iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_sender.inl

```diff
@@ -151,19 +150,8 @@ inline bool ChunkSender<ChunkSenderDataType>::getChunkReadyForSend(const mepoo::
 {
     if (getMembers()->m_chunksInUse.remove(chunkHeader, chunk))
     {
-        auto& chunkInfo = chunk.getChunkHeader()->m_info;
-        if (!chunkInfo.m_externalSequenceNumber_bl)
-        {
-            // if the sequence number is NOT set by the user, we take the one from the chunk sender for the chunk info
-            chunkInfo.m_sequenceNumber = getMembers()->m_sequenceNumber;
-            getMembers()->m_sequenceNumber++;
-        }
-        else
-        {
-            // if the seqence number in the chunk info is set by the user, we still increment the internal one as this
-            // might be needed by midddleware spcific evaluation (like in introspection)
-            getMembers()->m_sequenceNumber++;
-        }
+        chunk.getChunkHeader()->m_sequenceNumber = getMembers()->m_sequenceNumber;
+        getMembers()->m_sequenceNumber++;
```

Comment:
Why post-increment when you are not using the previous value? Could you please use the pre increment operator.

Reply:
good question. legacy code 😁

Reply:
if there are no other issue, are you okay to fix this with the next PR?

Reply:
done